Model/Config.php

```diff
@@ -440,4 +440,12 @@ public function isOrderStatusChangedAdminEmailEnabled(): bool
     {
         return $this->scopeConfig->isSetFlag(self::XML_CONFIG_PATH_ORDER_STATUS_CHANGED_ADMIN_EMAIL, ScopeInterface::SCOPE_WEBSITE);
     }
+
+    /**
+     * @return bool
+     */
+    public function isQuoteOverrideEnabled(): bool
+    {
+        return $this->scopeConfig->isSetFlag(self::XML_CONFIG_PATH_QUOTE_OVERRIDE, ScopeInterface::SCOPE_WEBSITE);
```

Comment:
The config for this is set to only be shown in global, so we can omit the scope flag here:
```suggestion
        return $this->scopeConfig->isSetFlag(self::XML_CONFIG_PATH_QUOTE_OVERRIDE);
```